addons/mail/static/src/components/message/message.js

```diff
@@ -549,8 +554,13 @@ class Message extends Component {
      * @param {MouseEvent} ev
      */
     _onClickReply(ev) {
-        ev.stopPropagation();
-        this.message.replyTo();
+        // Use this._wasSelected because this.props.isSelected might be changed
+        // by the parent global click handler before this method is executed.
```

Comment:
"[...] parent global **capture** click [...]" (easier to spot that this comment is a workaround of an already workaround by using capture mode)

Reply:
ok, reworded everything as the "parent" part wasn't correct anymore either